pkg/controller/volume/pvcprotection/pvc_protection_controller.go

```diff
@@ -176,6 +182,11 @@ func (c *Controller) processPVC(pvcNamespace, pvcName string) error {
 }
 
 func (c *Controller) addFinalizer(pvc *v1.PersistentVolumeClaim) error {
+	// Skip adding Finalizer in case the PVC Protection feature is not enabled
```

Comment:
I think I am missing something but which bits will by default remove the finalizer if feature gate is disabled?  I do not see that in the code.

Reply:
I think what we need to add is - when feature gate is disabled then we need to return `false` by default from `isBeingUsed` function so as controller can run in "remove finalizer" only mode. As it exists - this code will not remove the finalizer from PVCs which are in use by pods, even if feature gate is disabled.